src/backend/curses/pan.rs

```diff
@@ -0,0 +1,218 @@
+                // TODO: not sure what those are
+                pancurses::Input::KeySReset => Event::Refresh,
+                pancurses::Input::KeyReset => Event::Refresh,
+                pancurses::Input::KeyPrint => Event::Refresh,
+                pancurses::Input::KeyLL => Event::Refresh,
+                pancurses::Input::KeyAbort => Event::Refresh,
+                pancurses::Input::KeySHelp => Event::Refresh,
+                pancurses::Input::KeyLHelp => Event::Refresh,
+                pancurses::Input::KeyBTab => Event::Refresh,
+                pancurses::Input::KeyBeg => Event::Refresh,
+                pancurses::Input::KeyCancel => Event::Refresh,
+                pancurses::Input::KeyClose => Event::Refresh,
+                pancurses::Input::KeyCommand => Event::Refresh,
+                pancurses::Input::KeyCopy => Event::Refresh,
+                pancurses::Input::KeyCreate => Event::Refresh,
+                pancurses::Input::KeyEnd => Event::Refresh,
+                pancurses::Input::KeyExit => Event::Refresh,
+                pancurses::Input::KeyFind => Event::Refresh,
+                pancurses::Input::KeyHelp => Event::Refresh,
+                pancurses::Input::KeyMark => Event::Refresh,
+                pancurses::Input::KeyMessage => Event::Refresh,
+                pancurses::Input::KeyMove => Event::Refresh,
+                pancurses::Input::KeyNext => Event::Refresh,
+                pancurses::Input::KeyOpen => Event::Refresh,
+                pancurses::Input::KeyOptions => Event::Refresh,
+                pancurses::Input::KeyPrevious => Event::Refresh,
+                pancurses::Input::KeyRedo => Event::Refresh,
+                pancurses::Input::KeyReference => Event::Refresh,
+                pancurses::Input::KeyRefresh => Event::Refresh,
+                pancurses::Input::KeyReplace => Event::Refresh,
+                pancurses::Input::KeyRestart => Event::Refresh,
+                pancurses::Input::KeyResume => Event::Refresh,
+                pancurses::Input::KeySave => Event::Refresh,
+                pancurses::Input::KeySBeg => Event::Refresh,
+                pancurses::Input::KeySCancel => Event::Refresh,
+                pancurses::Input::KeySCommand => Event::Refresh,
+                pancurses::Input::KeySCopy => Event::Refresh,
+                pancurses::Input::KeySCreate => Event::Refresh,
+                pancurses::Input::KeySDC => Event::Refresh,
+                pancurses::Input::KeySDL => Event::Refresh,
+                pancurses::Input::KeySelect => Event::Refresh,
+                pancurses::Input::KeySEnd => Event::Refresh,
+                pancurses::Input::KeySEOL => Event::Refresh,
+                pancurses::Input::KeySExit => Event::Refresh,
+                pancurses::Input::KeySFind => Event::Refresh,
+                pancurses::Input::KeySHome => Event::Refresh,
+                pancurses::Input::KeySIC => Event::Refresh,
```

Comment:
I suppose `KeySIC` is `Shift-Ins`, but on my desktop it triggers "paste" instead.